Microsoft.OneWeek.Hack.Microids.MessageRouter/SingleLineConsoleLoggerProvider.cs

```diff
@@ -0,0 +1,177 @@
+using System;
```

Comment:
Shouldn't we make a common or lib folder/project to avoid duplicating code?

Reply:
That would be great, but we killed the "Core" library because it was causing confusion. So, yes it should, but for this test, we aren't going to change it anyway.